allenact/algorithms/onpolicy_sync/runner.py

```diff
@@ -50,16 +52,22 @@ def __init__(
         deterministic_cudnn: bool = False,
         deterministic_agents: bool = False,
         mp_ctx: Optional[BaseContext] = None,
-        multiprocessing_start_method: str = "forkserver",
+        multiprocessing_start_method: str = "default",
         extra_tag: str = "",
         disable_tensorboard: bool = False,
         disable_config_saving: bool = False,
     ):
         self.config = config
         self.output_dir = output_dir
         self.loaded_config_src_files = loaded_config_src_files
-        self.seed = seed
+        self.seed = seed if seed is not None else random.randint(0, 2 ** 31 - 1)
```

Comment:
this is great. then we ensure we always restart with the same (randomly chosen) seed. it was the last thing I was a bit annoyed with (that I could catch). nice! 👍